docs/apis/common/index.md

```diff
@@ -1350,3 +1350,397 @@ You may specify program arguments before the job is executed. The plan visualiza
 the execution plan before executing the Flink job.
 
 {% top %}
+
+Metrics
```

Comment:
I would make this a separate page under `Programming Guide`. It's guide hidden under the `Basic API` concepts although I understand that it it is technically the right place for it.